presto-main/src/main/java/com/facebook/presto/sql/planner/PlanFragmenter.java

```diff
@@ -208,6 +209,15 @@ else if (exchange.getType() == ExchangeNode.Type.REPARTITION) {
                     .map(PlanFragment::getId)
                     .collect(toImmutableList());
 
+            if (exchange.getOrderingScheme().isPresent()) {
+                return new MergeRemoteSourceNode(
+                        exchange.getId(),
+                        childrenIds,
+                        exchange.getOutputSymbols(),
+                        exchange.getOrderingScheme().get().getOrderBy(),
```

Comment:
The reason is to have ordering scheme only in `ExchangeNode`, similar to `PartitioningScheme`. I didn't want to introduce it everywhere.